--- a/node_modules/ol/proj.js	2018-10-14 10:25:13.101904148 +0200
+++ b/node_modules/ol/proj.js	2018-10-14 10:33:07.872292132 +0200
@@ -64,25 +64,6 @@
 import {add as addTransformFunc, clear as clearTransformFuncs, get as getTransformFunc} from './proj/transforms.js';
 
 
-/**
- * A projection as {@link module:ol/proj/Projection}, SRS identifier
- * string or undefined.
- * @typedef {module:ol/proj/Projection|string|undefined} ProjectionLike
- * @api
- */
-
-
-/**
- * A transform function accepts an array of input coordinate values, an optional
- * output array, and an optional dimension (default should be 2).  The function
- * transforms the input coordinate values, populates the output array, and
- * returns the output array.
- *
- * @typedef {function(Array<number>, Array<number>=, number=): Array<number>} TransformFunction
- * @api
- */
-
-
 export {METERS_PER_UNIT};
 
 export {Projection};
